response.go

```diff
@@ -156,7 +156,7 @@ func (r *Response) setReceivedAt() {
 }
 
 func (r *Response) fmtBodyString(sl int64) string {
-	if r.body != nil {
+	if len(r.body) > 0 {
```

Comment:
@haivp3010 Can you update this line as -
```go
if r.body != nil && len(r.body) > 0 {
```

Reply:
@jeevatkm same as above